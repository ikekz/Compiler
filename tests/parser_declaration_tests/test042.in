const
	c = 2.0;
type
	t = c;
begin