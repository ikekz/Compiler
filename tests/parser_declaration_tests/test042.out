(4,7) Error: Error in type definition